dashboard-widgets.el

```diff
@@ -342,6 +342,11 @@ ARGS should be a plist containing `:height', `:v-adjust', or `:face' properties.
   :type 'string
   :group 'dashboard)
 
+(defcustom dashboard-heading-shorcut-key-str " (%s)"
```

Comment:
Naming is hard, maybe a more descriptive name could be `dashboard-heading-shorcut-format`. This could be another PR.